server/sample.env

```diff
@@ -1,3 +1,3 @@
 JWT_SECRET=
 MONGO_URI=
-TEAM_NAMES=Bonnie,Sina, Vic, Kevin
\ No newline at end of file
+TEAM_NAMES=Bonnie,Sina, Vic, Kevin, Milad
```

Comment:
oh just make sure next time separate out your PRs as the description didn't mention this change.